AtReconstruction/AtFilter/AtSCACorrect.cxx

```diff
@@ -0,0 +1,60 @@
+#include "AtSCACorrect.h"
+
+#include "AtMap.h"
+#include "AtPad.h"
+#include "AtPadReference.h"
+#include "AtPadValue.h"
+#include "AtRawEvent.h"
+
+#include <FairLogger.h>
+
+#include <numeric>
+void AtSCACorrect::Filter(AtPad *pad, AtPadReference *padReference)
+{
+   removeBaseline(pad, padReference);
+   removePhase(pad, padReference);
+}
+
+void AtSCACorrect::removeBaseline(AtPad *pad, AtPadReference *padReference)
+{
+
+   AtPad *baselinePad;
```

Comment:
Not given an initial value, and is not given a valid address under every logical path